pkg/webhook/admission/pod/agent_injector.go

```diff
@@ -197,6 +202,13 @@ func (ag *AgentInjector) InjectAgent(pod *v1.Pod) error {
 			component,
 		}
 		args = append(args, loggerArgs...)
+
+		// Add TLS cert name if specified. If not specified it will fall back to the arg's default.
+		if ag.loggerConfig.CaCertFile != "" {
+			args = append(args, LoggerArgumentCaCertFile, ag.loggerConfig.CaCertFile)
+		}
+		// Whether to skip TLS verification. If not present in the ConfigMap, this will default to `false`
+		args = append(args, LoggerArgumentTlsSkipVerify, strconv.FormatBool(ag.loggerConfig.TlsSkipVerify))
```

Comment:
if the ca cert is not found, isn't it better to use true to skip the validation?